signer/src/error.rs

```diff
@@ -593,6 +593,13 @@ pub enum Error {
         /// Maximum sBTC mintable
         max_mintable: u64,
     },
+
+    /// An error which can be used in test code instead of `unimplemented!()` or
+    /// other alternatives, so that an an actual error is returned instead of
+    /// panicking.
+    #[cfg(test)]
+    #[error("Dummy (for testing purposes)")]
+    Dummy,
```

Comment:
mmm panicking if we hit some unimplemented test path could be better, otherwise we may "fail" silently since we handle the error in multiple places (I'm thinking like in the tx coordinator event loop) and not test what we actually want to test.

Reply:
Yeah I agree, it should be used correctly -- in this case it's used in the tests for the `info_handler()` for simulating an error from the mocked `*Interact` implementations, where I don't want it to panic, and I didn't really want to just choose a random unrelated error either.